benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -906,6 +920,24 @@ void GraphicsBenchmarkApp::DrawExtraInfo()
     uint64_t frequency = 0;
     GetGraphicsQueue()->GetTimestampFrequency(&frequency);
 
+    ImGui::Columns(2);
```

Comment:
`ImGui::Columns(2);` is only required once at the beginning, the other ones are not needed

Reply:
Done.